src/app/ApplicationTemplate.Access/NativePlatform/Connectivity/IConnectivityRepository.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+
+namespace ApplicationTemplate.DataAccess;
+
+/// <summary>
+/// Provides access to the current network state and detects changes.
+/// </summary>
+public interface IConnectivityRepository
```

Comment:
Is repository the correct suffix? Connectivity isn't really a data source, nor persisted? It's a mobile API.

I'm not sure, it's almost there I guess :)

Reply:
After discussions with JP we've come to the conclusion that its place should be in the Access layer (Access Layer (Repositories): Handles interactions with external systems (e.g., APIs, databases, native platform features)).

Reply:
And like you said, we could argue that Connectivity API return data about the connectivity state from the device's API